src/transactions/payment.ts

```diff
@@ -0,0 +1,28 @@
+import { Transaction, TransactionType } from '@algorandfoundation/algokit-transact'
+import { Address } from '@algorandfoundation/sdk'
+import { AlgoAmount } from '../types/amount'
+import { CommonTransactionParams, TransactionHeader } from './common'
+
+/** Parameters to define a payment transaction. */
+export type PaymentParams = CommonTransactionParams & {
+  /** The address of the account that will receive the Algo */
+  receiver: string | Address
+  /** Amount to send */
+  amount: AlgoAmount
+  /** If given, close the sender account and send the remaining balance to this address
+   *
+   * *Warning:* Be careful with this parameter as it can lead to loss of funds if not used correctly.
+   */
+  closeRemainderTo?: string | Address
```

Comment:
This field doesn't appear to be mapped?

Can you double check the mappings for all transaction types?